.github/workflows/linux-full-tests.yml

```diff
@@ -194,9 +194,5 @@ jobs:
     - name: Check global header
       run: |
         echo "#include <ql/quantlib.hpp>" > test1.cpp && echo "int main() { return 0; }" >> test1.cpp
-        echo "#include <ql/quantlib.hpp>" > test2.cpp
-        if test ${{ matrix.shortname }} = sessions ; then
-            echo "namespace QuantLib { ThreadKey sessionId() { return 0; } }" >> test2.cpp
-        fi
         make install
-        ${{ matrix.cxx }} -O2 -g0 -Wall -Wno-unknown-pragmas -Werror ${{ matrix.cxxflags }} `quantlib-config --cflags` test1.cpp test2.cpp `quantlib-config --libs`
+        ${{ matrix.cxx }} -O2 -g0 -Wall -Wno-unknown-pragmas -Werror ${{ matrix.cxxflags }} `quantlib-config --cflags` test1.cpp `quantlib-config --libs`
```

Comment:
You need to keep `test2.cpp` in there, you only need to remove the `sessionId` declaration — the purpose here was not to check `sessionId()`, it is to link together two files that both include `quantlib.hpp` in order to check that there's no duplicated function causing the linker to fail (for example because we forgot an `inline` in a header file).